### PR TITLE
[POZ-2020] Initial commit for Poznan / Poland 2020 :)

### PR DESCRIPTION
**Pull request by mazek**

### CODE REVIEW

File: data/events/2020-poznan.yml [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    employer: "Mirantis"
+    twitter: "eftepede"
+    image: bartek-stalewski.jpg
+  - name: "Klaudia Jasniak"
+    employer: "JCommerce"
+    linkedin: "https://www.linkedin.com/in/klaudia-ja%C5%9Bniak-97283180/"
+    image: klaudia-jasniak.jpg
+
+
+organizer_email: "organizers-poznan-2019@devopsdays.org" # Put your organizer email address here
```
Comment (yvovandoorn) [inline data/events/2020-poznan.yml]: Format for emails has changed! It is now city@devopsdays.org. Please address.

File: data/events/2020-poznan.yml [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+proposal_email: "proposals-poznan-2019@devopsdays.org" # Put your proposal email address here
```
Comment (yvovandoorn) [inline data/events/2020-poznan.yml]: Format for emails has changed! It is now city@devopsdays.org. Please address.

File: data/events/2020-poznan.yml [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+  - id: bronze
+    label: Bronze
+  - id: community
+    label: Community
+  - id: socialevent
+    label: Social Event
+  - id: workshop
+    label: Workshop
+
+program:
```
Comment (yvovandoorn) [inline data/events/2020-poznan.yml]: I dont think you want to bring in the old program!

**Comment by yvovandoorn** [comment]
Hi @mazek 

Some things need to be addressed here:
- emails need to be fixed to city@devopsdays.org format
- email to info@devopsdays.org with current organizers should be sent
- you dragged in your 2019 program into this. I don't think you want to do that :-).

File: data/events/2020-poznan.yml (hunk 2 above)
Comment (mazek) [reply data/events/2020-poznan.yml]: Corrected that, thx

File: data/events/2020-poznan.yml (hunk 3 above)
Comment (mazek) [reply data/events/2020-poznan.yml]: It's commented out for just future reference, but for clarity I removed it. thx

File: data/events/2020-poznan.yml (hunk 1 above)
Comment (mazek) [reply data/events/2020-poznan.yml]: Corrected that, thx

**Comment by yvovandoorn** [comment]
Almost there @mazek! I still see a bunch of 2019 speaker photos being part of the commit. 

Once those are no longer part of the commit, we can merge.

**Pull request merged**